examples/pytorch/language-modeling/run_clm.py

```diff
@@ -507,7 +547,11 @@ def main():
                 xs.mark_sharding(param, spmd_mesh, (('tensor', 'fsdp'), None))  # keep this fsdp.
 
             print(f'{name} {torch_xla._XLAC._get_xla_sharding_spec(param)}')
+            torch_xla.sync()
+            dict_of_params[name] = param
 
+        model.load_state_dict(dict_of_params, assign=True)
+        model.to('xla')
```

Comment:
Question: with meta device, did we skip loading the model into CPU RAM but loading into `XLA` device directly?